imageio/plugins/ffmpeg.py

```diff
@@ -91,33 +115,39 @@ def get_exe():
         try:
             with open(os.devnull, "w") as null:
                 sp.check_call([exe, "-version"], stdout=null, stderr=sp.STDOUT)
-                return exe
+                exes.append(exe)
```

Comment:
see https://github.com/imageio/imageio/pull/270/commits/956955ef187aeed6fe3f6839b41c9324e4c105cd: I added the `imageio_remove_bin` entry point.